README.md

```diff
@@ -20,14 +20,38 @@ The API uses [Django REST framework](http://www.django-rest-framework.org/)
 
 
 ## Installation - Git and repository setup
-
+### Setting up your Git user and cloning the repo
 ```bash
 $ git config --global core.autocrlf false
 $ git config --global user.name "<your github username>"
 $ git config --global user.email your.github@email.com
 $ git clone git@github.com:dotkom/onlineweb4.git
 $ cd onlineweb4
 ```
+### Setting up the onlineweb4
+Download dependencies by running
+```bash
+make
+```
+
+Lets apply all the django changes to the database by running
+```bash
+# creates migrations by packaging model changes into individual migration files
+make makemigrations
```

Comment:
Is `make makemigrations` needed? All migrations are in the repository and will be applied by `make migrations`, won't it?

Reply:
Might be tree! I'll test if it does and then change it